ambry-store/src/test/java/com.github.ambry.store/StorageManagerTest.java

```diff
@@ -107,7 +122,10 @@ public void storeStartFailureTest() throws Exception {
     for (Integer badReplicaIndex : badReplicaIndexes) {
       new File(replicas.get(badReplicaIndex).getReplicaPath()).setReadable(false);
     }
-    StorageManager storageManager = createAndStartStoreManager(replicas);
+    StorageManager storageManager = createAndStartStoreManager(replicas, metricRegistry);
+    Map<String, Counter> counters = metricRegistry.getCounters();
+    assertEquals(2, getCounterValue(counters, DiskManager.class.getName(), "TotalStoreStartFailures"));
```

Comment:
Minor: Could you use `badReplicaIndexes.size()` instead of 2 here? Just in case we change the setup of the test.